engines/sludge/objtypes.cpp

```diff
@@ -47,6 +47,17 @@ void ObjectManager::kill() {
 	_allObjectTypes.clear();
 }
 
+ObjectType *ObjectManager::debugFindObjectType(int i) {
```

Comment:
What is the difference between this one and `findObjectType(int i)`? You are still loading it in both cases.

I do not like this design in general. Why not adding an optional parameter to `findObjectType(int i, bool skipLoad = false)` and not load it if it is not present in `_allObjectTypes`?